rt-pwa-extensions.php

```diff
@@ -69,3 +69,7 @@ function( $headers ) {
 		return $headers;
 	}
 );
+
+if ( in_array( 'izooto-web-push/izooto.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ), true ) ) {
```

Comment:
Use `is_plugin_active()` function instead of this. For that you also need to include `wp-admin/includes/plugin.php` if function does not exists.